bodhi/models/models.py

```diff
@@ -1696,7 +1696,7 @@ def met_testing_requirements(self):
             if comment.user.name == 'bodhi' and \
                comment.text.startswith('This update has reached') and \
                comment.text.endswith('days in testing and can be pushed to'
-                                     ' stable now if the maintainer wishes'):
+                                     ' stable now if the maintainer wishes.'):
```

Comment:
This will be problematic.

We need to be able to match both the old form without the ``.`` and the new form with it.

Maybe change it to ``' stable now if the maintainer wishes' in comment.text``?